align/schema/constraint.py

```diff
@@ -529,6 +529,12 @@ def place_symmetric_instances_validator(cls, value):
         return value
 
 
+class AlignGroup(SoftConstraint):
```

Comment:
`AlignGroup` could be confusing given `Align*` and `GroupBlocks` constraints. How about rename to `CompactPlacement` or `ConfigurePlacement` that we can extend in the future with other configuration parameters?

Reply:
Sure. I'll rename the constraint to `CompactPlacement`.